transport-native-io_uring/src/test/java/io/netty/incubator/channel/uring/IOUringEventLoopTest.java

```diff
@@ -50,13 +63,43 @@ protected Class<? extends ServerChannel> serverChannelClass() {
         return IOUringServerSocketChannel.class;
     }
 
-    @Test
+//    @Test
     public void shutdownGracefullyZeroQuietBeforeStart() throws Exception {
         EventLoopGroup group = newEventLoopGroup();
         assertTrue(group.shutdownGracefully(0L, 2L, TimeUnit.SECONDS).await(1500L));
     }
 
-    @Test
+    @RepeatedTest(1000)
+    public void shutdownNonGraceful() throws Exception {
+        EventLoopGroup group = newEventLoopGroup();
+        CountDownLatch latch = new CountDownLatch(1);
+        group.submit(() -> latch.countDown());
+        latch.await(5, TimeUnit.SECONDS);
+        if (!group.shutdownGracefully(0L, 0L, TimeUnit.NANOSECONDS).await(2000L)) {
+            dumpThreads();
+        }
+        // Fails with
+        // [ERROR] Tests run: 1010, Failures: 0, Errors: 1, Skipped: 3, Time elapsed: 4.527 s <<< FAILURE! - in io.netty.incubator.channel.uring.IOUringEventLoopTest
+        // [ERROR] shutdownNonGraceful[301]  Time elapsed: 2.009 s  <<< ERROR!
+        // java.lang.Exception:
+        // What IO threads doing? Stack traces:
+        // Thread io_uring-0:
+        //		io.netty.incubator.channel.uring.Native.ioUringEnter(Native Method)
+        //		io.netty.incubator.channel.uring.IOUringCompletionQueue.ioUringWaitCqe(IOUringCompletionQueue.java:99)
+        //		io.netty.incubator.channel.uring.IOUringEventLoop.cleanup(IOUringEventLoop.java:338)
+        //		io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:1044)
+        //		io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74)
+        //		java.lang.Thread.run(Thread.java:750)
+        //
+        //	at io.netty.incubator.channel.uring.IOUringEventLoopTest.dumpThreads(IOUringEventLoopTest.java:162)
+        //	at io.netty.incubator.channel.uring.IOUringEventLoopTest.shutdownNonGraceful(IOUringEventLoopTest.java:80)
+        //	at sun.reflect.GeneratedMethodAccessor2.invoke(Unknown Source)
+        //	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
+        //	at java.lang.reflect.Method.invoke(Method.java:498)
+        //  ...
```

Comment:
After adding all the logging, loops that failed to shutdown would look like
```
19:18:01.044 [main] DEBUG i.n.i.channel.uring.IOUringEventLoop - wakeup (false) - 658532887
19:18:01.044 [main] DEBUG i.n.i.channel.uring.IOUringEventLoop - wakeup (true) - 658532887
19:18:01.044 [io_uring-315] DEBUG i.n.i.channel.uring.IOUringEventLoop - Setting pendingWakeup - 658532887
19:18:01.044 [io_uring-315] DEBUG i.n.i.channel.uring.IOUringEventLoop - Cleanup called, 658532887
```

whereas a loop that successfully shutdown (and received a wakup) would log like this:
```
19:18:03.073 [main] DEBUG i.n.i.channel.uring.IOUringEventLoop - wakeup (false) - 593415583
19:18:03.076 [main] DEBUG i.n.i.channel.uring.IOUringEventLoop - wakeup (true) - 593415583
19:18:03.076 [io_uring-318] DEBUG i.n.i.channel.uring.IOUringEventLoop - submitAndWait 593415583
19:18:03.076 [io_uring-318] DEBUG i.n.i.channel.uring.IOUringEventLoop - Setting pendingWakeup - 593415583
19:18:03.080 [io_uring-318] DEBUG i.n.i.channel.uring.IOUringEventLoop - Event fd: 81 - 593415583
19:18:03.080 [io_uring-318] DEBUG i.n.i.channel.uring.IOUringEventLoop - Clearing pendingWakeup (handle) - 593415583
19:18:03.081 [io_uring-318] DEBUG i.n.i.channel.uring.IOUringEventLoop - Cleanup called, 593415583
```